metrics/ci_integrations/lib/destination/firestore/factory/firestore_factory.dart

```diff
@@ -0,0 +1,29 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'package:ci_integration/client/firestore/firestore.dart' as fs;
+import 'package:firedart/firedart.dart';
```

Comment:
```suggestion
import 'package:ci_integration/client/firestore/firestore.dart';
import 'package:firedart/firedart.dart' as fd;
```
Does it make sense to make the `firedart` import named instead of `firestore` to make documentation point to the right classes?